app/experimenter/experiments/models/nimbus.py

```diff
@@ -195,6 +195,24 @@ def __str__(self):
         return self.name
 
 
+class NimbusDocumentationLink(models.Model):
+    experiment = models.ForeignKey(
+        NimbusExperiment,
+        related_name="documentation_links",
+        on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=255, null=False)
+    link = models.URLField(max_length=255, null=False)
+
+    class Meta:
+        verbose_name = "Nimbus Documentation Link"
+        verbose_name_plural = "Nimbus Documentation Links"
+        ordering = ("id",)
+
+    def __str__(self):
+        return "%s (%s)" % (self.title, self.link)
```

Comment:
In general [fstrings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) are preferred for formatting now.